packages/staking/src/features/drawer/preferences/NoPoolsSelected.tsx

```diff
@@ -0,0 +1,23 @@
+import { Button, Text } from '@lace/ui';
+import { useTranslation } from 'react-i18next';
+import Empty from './empty.svg';
+import * as styles from './NoPoolsSelected.css';
+
+export const NoPoolsSelected = ({
+  onBrowsePoolsButtonClick,
+}: {
+  onBrowsePoolsButtonClick: () => void;
+}): React.ReactElement => {
+  const { t } = useTranslation();
+  return (
+    <div className={styles.container}>
+      <Empty className={styles.icon} />
+      <Text.Body.Normal className={styles.text} weight="$semibold">
+        {t('drawer.preferences.noSelectedPools')}
+      </Text.Body.Normal>
+      <div className={styles.button}>
+        <Button.CallToAction label={t('drawer.preferences.browsePools')} onClick={onBrowsePoolsButtonClick} />
+      </div>
+    </div>
```

Comment:
nitpick - I'd consider using the ui component `Box` which leverages the `@lace/ui` framework and I'd consider dropping the button style as it seems overall the preferred way in the codebase to use the `mt` prop. Moreover, I'd consider moving the "weight" property of the `Text.Body.Normal` within the styles to keep the styles for the respective component in a single place

```suggestion
    <Box className={styles.container}>
      <Empty className={styles.icon} />
      <Text.Body.Normal className={styles.text}>
        {t('drawer.preferences.noSelectedPools')}
      </Text.Body.Normal>
      <Box mt="$40">
        <Button.CallToAction label={t('drawer.preferences.browsePools')} onClick={onBrowsePoolsButtonClick} />
      </Box>
    </Box>
```